integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java

```diff
@@ -246,6 +250,11 @@ public void testSampleDomainInPv(String model) {
 
     // run create-domain.sh to create domain.yaml file, run kubectl to create the domain and verify
     createDomainAndVerify(domainName, sampleBase);
+
+    if (script.equals("wdt")) {
+      copyModelFileForUpdateDomain(sampleBase);
+      updateDomainAndVerify(domainName, sampleBase, domainNamespace);
```

Comment:
Can you update the javadoc with the fact that, in case of wdt model, it also update the domain using the  sample script update-domain.sh

Reply:
done